packages/phone-number/lib/parse/index.js

```diff
@@ -19,56 +20,49 @@ export default function parse({ input, countryCode = 'US' }) {
     extension: null,
 
   };
-  const cleanInput = (input || '').replace(cleanRegex, '');
-  const startWithPlus = cleanInput[0] === '+';
-  const withoutPlus = cleanInput.replace(plusRegex, '');
-  const startWithStar = withoutPlus[0] === '*';
-  if (startWithPlus && startWithStar) {
+
+  var phoneNumberIsValid = phoneNumber => phoneNumber.length > 3 && validRegExp.test(phoneNumber);
```

Comment:
Please do not use "var" anymore, use const/let to declare variables.

Reply:
Secondly, why are you using arrow functions to declare a function within another function if scopes are not even used?
A simple function declaration on top of the file should be sufficient. Essentially you are creating multiple copies of one time use functions that do exactly the same thing, it's not a good pattern and it waste memory.
```javascript
function phoneNumberIsValid(phoneNumber) { 
  //...content 
}
```

Reply:
Lastly, why do you want to make this into a function when this check is only used once?
It is a very simple conditional statement that is not being re-used. Please don't create functions like this if you are not planning to re-use it somewhere else.

Reply:
IMHO, it may be fine to put complicated logic into a single function which is just for a more readable code? But in this case, since the wrapped code is indeed atomic, might not suit for wrap them into function just in order to give it a name.

Reply:
Indeed readability is a valid argument. However, please consider the following 2 cases:
```javascript
class A {
  shortConditionalFn() {
    // a short comment should be enough to make it readable
    if (
      "case 1" > 0 ||
      "case 2" >= 3
    ) {
      // do something
    } 
  }
  longConditionalFn() {
    if (easyToReadFn()) {
      // do something else
    }
  }
}

function easyToReadFn() {
  return !!(
    "lots" > 0 ||
    "of" > 1 ||
    "conditions" > 2
  );
}
```

For a short and one-time-use conditional, it is cleaner to just have a short comment to make it easy to understand. Where as for very long and complicated conditionals, readability is a valid reason to wrap it in a function.